source/core/GenericFunctions.cpp

```diff
@@ -668,7 +668,36 @@ InstructionCore* EmitMaxMinEltsInstruction(ClumpParseState* pInstructionBuilder)
     }
     return pInstruction;
 }
+//------------------------------------------------------------
+bool GetMinimumArrayDimensions(TypedArrayCoreRef sourceArray1, TypedArrayCoreRef sourceArray2,
```

Comment:
That would be huge list, as most prims are either unops or binops, and which is which is pretty obvious, as is whether the arguments are scalar or vector deciding which of these wrappers is used.
What I think does needs to be documented are the exceptions and corner cases, such as unop/binops with two outputs.